src/Summoner/Template/Cabal.hs

```diff
@@ -97,9 +97,9 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
           hs-source-dirs:      app
           main-is:             Main.hs
 
-          build-depends:       $settingsBase
-                             $commaRepo
-                             $commaPreludeLibrary
+          build-depends:       $settingsBaseType
+                               $commaRepo
+                               $commaPreludeLibrary
```

Comment:
The same here. These are line with commas in it. So, no need it two extra spaces.